impl/src/context.rs

```diff
@@ -69,7 +69,6 @@ pub struct CrateContext {
   pub targets: Vec<BuildableTarget>,
   pub build_script_target: Option<BuildableTarget>,
   pub source_details: SourceDetails,
-  pub sha256: Option<String>,
```

Comment:
kornholi@ commented on the utility of this field here https://github.com/google/cargo-raze/pull/149#issuecomment-621493706 .

I actually forgot what this was for and I want to revisit this. I would be sad to see it deleted without remembering what it was for. No action item from you yet, just mostly a reminder for me to figure out what's going on here.

Reply:
Ack - this is preserved in the most recent update to #149 , where the value is retrieved from the Lockfile.